CHANGELOG_PENDING.md

```diff
@@ -41,6 +41,9 @@
 - [automation/dotnet] Expose WorkspaceStack.GetOutputsAsync
   [#6699](https://github.com/pulumi/pulumi/pull/6699)
 
+- [automation/dotnet] Expose WorkspaceStack.(Import,Export)StackAsync and StackDeployment helper class
```

Comment:
Since the methods are on both LocalWorkspace and WorkspaceStack maybe something like the following?

```suggestion
- [automation/dotnet] Implement (Import,Export)StackAsync methods on LocalWorkspace and WorkspaceStack and expose StackDeployment helper class.
```